mysql-test/suite/rocksdb_rpl/r/rpl_rocksdb_extra_col_master.result

```diff
@@ -429,6 +429,8 @@ include/check_slave_is_running.inc
 ** Stop and Reset Slave **
 
 STOP SLAVE;
+Warnings:
```

Comment:
Here and everywhere in this file
Probably it's better to change the command in the .test file

Reply:
Explained